compiler-utils/src/main/java/com/squareup/anvil/compiler/internal/PsiUtils.kt

```diff
@@ -768,6 +769,8 @@ public fun FqName.requireClassDescriptor(module: ModuleDescriptor): ClassDescrip
 @ExperimentalAnvilApi
 public fun FqName.classDescriptorOrNull(module: ModuleDescriptor): ClassDescriptor? {
   return module.resolveClassByFqName(this, FROM_BACKEND)
+    // returns the original reference being pointed to by a typealias
```

Comment:
That's okay for now, but we prefer full sentences as comments starting with an uppercase letter and ending with punctuation mark.

Reply:
Thanks for the explanation.  I'd noticed that the project leaned that way, but didn't realize that it'd been formalized.